### DISCUSSION TITLE
attaching BepInEx to a game that is only started via launcher

### DISCUSSION THREAD

**Discussion by pavlexander** [BepInEx/BepInEx#712] [Help and Q&A]
I am trying add `BepInEx` to a game that is only launched through the launcher. I.e. raid shadow legends. It's only working when started by a launcher. If you run the game by directly invoking the `raid.exe` then it never loads.. The BepInEx does not seem to support this use-case. If anyone managed to attach `BepInEx` to a Raid instance I would be interested to know how it's done..

I have tried extracting the command line arguments of raid.exe process (`"C:\RAID\PlariumPlay\StandAloneApps\raid-shadow-legends\73943\Raid.exe"  -locale:en -socketAddress:127.0.0.1:5201 -salt:xxx= -resource-folder:"C:\RAID\PlariumPlay\StandAloneApps\raid-shadow-legends\resources" -logFile "C:\RAID\PlariumPlay\StandAloneApps\raid-shadow-legends\73943\log.txt"`), then creating the raid shortcut with the same parameters, but it does not seem to solve the problem.

My next step would be to inspect the game though the Unity Explorer but that's after the `BepInEx` starts working property..

tried v5.4.22 and BepInEx 6.0.0-pre.1

**Comment by toebeann** [comment]
RAID: Shadow Legends is a mobile game which you are playing via an emulator. BepInEx is not designed to work in that situation.

**Comment by bnthomason** [comment]
Is there a work around? I want to access the camera controls and modify some assets. It's made with unity so I think it should work.

**Reply by toebeann** [reply]
No, because BepInEx doesn't support android builds.
